docs/thermometer-skills.md

```diff
@@ -45,6 +45,11 @@ aea install
 aea config set agent.default_connection fetchai/oef:0.2.0
 ```
 
+Alternatively, you can fetch the entire AEA:
+``` bash
+aea fetch fetchai/thermometer_aea:0.1.0 --alias my_thermometer_aea
```

Comment:
Doesn't one have to `aea install` after `aea fetch`, or does `fetch` take care of installing the dependencies as well? If the former, then add `aea install` after this line.